ai_traineree/agents/rainbow.py

```diff
@@ -83,7 +86,7 @@ def __init__(
 
         v_min = float(self._register_param(kwargs, "v_min", -10))
         v_max = float(self._register_param(kwargs, "v_max", 10))
-        self.num_atoms = int(self._register_param(kwargs, "num_atoms", 21))
+        self.num_atoms = int(self._register_param(kwargs, "num_atoms", 21, drop=True))
```

Comment:
If the value is not removed from the kwargs, the RainbowNet constructor fails as it gets twice the *num_atoms* parameter.